backend/src/test/java/com/huellapositiva/api/ProposalControllerShould.java

```diff
@@ -550,12 +543,16 @@ void return_200_when_submitting_revision_as_revisor_has_feedback_but_not_feedbac
                 .contentType(MediaType.APPLICATION_JSON)
                 .accept(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk());
+
+        JpaProposal jpaProposalWithStatusChanged = jpaProposalRepository.findByNaturalId(proposalId)
+                .orElseThrow(() -> new UserNotFoundException("Proposal not found. Account ID: " + proposalId));
+        assertThat(jpaProposalWithStatusChanged.getStatus().getId()).isEqualTo(2);
```

Comment:
Instead of that magic number, we should get the id from the ProposalStatus enum. That way, if we need to change the order in the future, we will still compare this to CHANGES_REQUESTED, and its easier to understand.

Reply:
It's done.